### PR TITLE
FIX: Pull to refresh

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

File: screen/wallets/list.js [hunk 1]
```diff
@@ -366,11 +366,7 @@ const WalletsList = () => {
     refreshTransactions(true, false);
   };
 
-  const refreshProps = isDesktop
-    ? {}
-    : {
-        ...(isElectrumDisabled ? {} : { refreshing: isLoading, onRefresh }),
-      };
+  const refreshProps = isDesktop || isElectrumDisabled ? {} : { refreshing: isLoading, onRefresh };
```
Comment (Overtorment) [inline screen/wallets/list.js:369]: A bit confusing line. Would be nice if such confusing lines in code always ha comments

**Review by Overtorment — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/m26ebjenwyzsnzj5chy4adl6ni?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/479)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/i4migxihq2wsbkxoouu6xdbngm?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/480)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/azy7s6bnpuhkbmbodiflbwpxmq?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/481)

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**